static/templates/pm_list_item.hbs

```diff
@@ -1,4 +1,4 @@
-<li class='{{#if is_active}}active-sub-filter{{/if}} {{#if is_zero}}zero-pm-unreads{{/if}} top_left_row expanded_private_message' data-user-ids-string='{{user_ids_string}}'>
+<li class='{{#if is_active}}active-sub-filter{{/if}} {{#if is_zero}}zero-pm-unreads{{/if}} expanded_private_message' data-user-ids-string='{{user_ids_string}}'>
```

Comment:
Should this be gaining the `bottom_left_row` class? It appears that just controls highlight behavior on hover, but as best I can tell, every item in the sidebar should have one or the other.

Reply:
Ahh, i wasn't aware of this, Huge Thanks!
I'll update the PR and push the changes.

Reply:
Updated !
Thanks!